config/environments/production.rb

```diff
@@ -49,17 +47,15 @@
   config.log_level = :debug
 
   # Prepend all log lines with the following tags.
-  # config.log_tags = [ :subdomain, :uuid ]
-
-  # Use a different logger for distributed setups.
-  # config.logger = ActiveSupport::TaggedLogging.new(SyslogLogger.new)
-  config.logger =  Logger.new(STDOUT)
```

Comment:
Moving this to the `if ENV["RAILS_LOG_TO_STDOUT"].present?` block below will disable logging if `RAILS_LOG_TO_STDOUT` is not set in the template. Either revert this change or set the env variable.